### PR TITLE
Gal 403/clinic info lsoa table

### PR DESCRIPTION
**Pull request by zainm97**
## Description
https://nhsd-jira.digital.nhs.uk/browse/GAL-403

This PR adds functionality to pass data to the front end to render LSOA table data. It uses the queryStringParameters of Postcode and miles to identify the LSOAs in the selected distance of the Phlebotomy site. It then passes the list of identified LSOA codes to another Lambda to return the number of eligible participants in each LSOA. The data is combined.

This PR also repurposes the LSOA table into the Postcode Table. The PR also adds a table called UniqueLsoas which harbours unique LSOAs along with associated data at an LSOA level

This ticket assumes that:

- UniqueLsoa table is populated with entries that include columns for average easting and northing. (Ticket 473 https://nhsd-jira.digital.nhs.uk/browse/GAL-473)
- And that the population database has an attribute for what LSOA code they reside in.

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: scripts/pipeline/unit_tests/test_nonprod_unique_lsoa_load.py [hunk 1]
```diff
@@ -1,63 +1,64 @@
-from scripts.pipeline.nonprod_lsoa_load.nonprod_lsoa_load import format_dynamodb_json
+import json
```
Comment (Abdul-NHS) [inline scripts/pipeline/unit_tests/test_nonprod_unique_lsoa_load.py:1]: Is this import used?

File: terraform/environment/dev/resources/lambda/getLsoaParticipants/lambdaHandler/getLsoaParticipantsLambda.js
Comment (Abdul-NHS) [inline terraform/environment/dev/resources/lambda/getLsoaParticipants/lambdaHandler/getLsoaParticipantsLambda.js:1]: I think this file structure has changed and should only live in src/lambdaName/lambda.js (example)

File: terraform/environment/dev/resources/lambda/getLsoaInRange/lambdaHandler/getLsoaInRangeLambda.js
Comment (Abdul-NHS) [inline terraform/environment/dev/resources/lambda/getLsoaInRange/lambdaHandler/getLsoaInRangeLambda.js:1]: This file also can be removed, should live in src which you already have

File: scripts/generatePersonTestData/README.txt
Comment (zainm97) [inline scripts/generatePersonTestData/README.txt:1]: GAL-211: Don't need to review in this PR. Has already been approved

File: scripts/generatePersonTestData/generatePersonTestData.js
Comment (zainm97) [inline scripts/generatePersonTestData/generatePersonTestData.js:1]: GAL-211: Don't need to review in this PR. Has already been approved

File: scripts/generatePersonTestData/package-lock.json
Comment (zainm97) [inline scripts/generatePersonTestData/package-lock.json:1]: GAL-211: Don't need to review in this PR. Has already been approved

**Review by Abdul-NHS — DISMISSED** [review]

**Review by Abdul-NHS — DISMISSED** [review]

File: scripts/pipeline/unit_tests/test_nonprod_unique_lsoa_load.py (hunk 1 above)
Comment (zainm97) [reply scripts/pipeline/unit_tests/test_nonprod_unique_lsoa_load.py:1]: good spot removing

**Review by Abdul-NHS — APPROVED** [review]
lgtm

**Review by Amar-NHS — APPROVED** [review]
As discussed, will review the DynamoDB client being created inside the handler in future PR. Approved!

**Pull request merged**